wgpu/src/lib.rs

```diff
@@ -2764,6 +2764,12 @@ impl Device {
                 )
         }
     }
+
+    /// Trigger the "lose the device" steps, which makes the Device invalid
+    /// and prevents further operations from succeeding.
+    pub fn lose(&self) {
+        self.context.device_lose(&self.id, self.data.as_ref())
+    }
```

Comment:
as commented above, we should definitely not expose this to the user-interface if we don't absolutely have to.
If we do it should come with a lot more documentation on what this is, why it's there, how it's not implemented on the web, how it relates to `destroy` etc.

Reply:
Agreed, and now that this patch will stub in destroy, it's no longer useful to be able to call "device.lose" at all.